.github/workflows/test.yml

```diff
@@ -70,6 +70,10 @@ jobs:
           - '1.2.*'
           - '1.3.*'
           - '1.4.*'
+          - '1.5.*'
```

Comment:
Perhaps we could also remove some of the older versions, but let's discuss first.

Reply:
Yeah, I think we can reduce it to the latest 3 minor versions 👍